schema/artwork/index.js

```diff
@@ -268,7 +268,7 @@ export const artworkFields = () => {
             .then(([sale, saleArtwork]) => {
               if (!sale) return false
 
-              return saleArtwork.bidder_positions_count < 1
+              return saleArtwork.reserve_status !== "reserve_met"
```

Comment:
I think so right? You may not have a reserve but if there is a buy now price that the merchant wants to sell then we should allow that.

I know BuyNow prices are usually set at reserve but i can see it being useful even when we don't have a reserve